tests/data/test_dataloader.py

```diff
@@ -441,12 +486,13 @@ def test_finetuning_dataloader_custom_split(tmp_path: pathlib.Path, split: str):
 
 
 def mock_get_file(path: str, destination: str, overwrite: bool = False):
-    make_tiny_ft_dataset(path=destination, size=16)
+    if Path(destination).suffix == '.jsonl':
```

Comment:
is this supposed to silently do nothing if not jsonl?

Reply:
ah, I think it should ideally throw a FileNotFoundError